editor.planx.uk/src/pages/FlowEditor/components/Subscription/Subscription.tsx

```diff
@@ -0,0 +1,25 @@
+import Container from "@mui/material/Container";
+import Typography from "@mui/material/Typography";
+import React from "react";
+import SettingsSection from "ui/editor/SettingsSection";
+
+import { Discount } from "./components/Discount";
+import { ServiceCharges } from "./components/ServiceCharges";
+import { SubscriptionProps } from "./types";
+
+export const Subscription = ({ serviceCharges }: SubscriptionProps) => (
+  <Container maxWidth="formWrap">
+    <SettingsSection>
+      <Typography variant="h2" component="h3" gutterBottom>
+        Subscription
+      </Typography>
+      <Typography variant="body1">
+        Details about your Plan✕ software subscription.
+      </Typography>
+    </SettingsSection>
+    <ServiceCharges serviceCharges={serviceCharges} />
+    {/* <Contract /> */}
+    <Discount serviceCharges={serviceCharges} />
+    {/* <SSLRenewal /> */}
```

Comment:
I've stubbed out `Contract` and `SSL` components, but they aren't "active" yet until we decide how to integrate data points from our Notion CRM into the editor - let's leave these for a follow-up PR !